tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
+  "include": [
+    "./src/**/*"
```

Comment:
Is this too permissive? Still not clear what we need here? I'm assuming this works together with the compile command?